client/src/app/services/user.service.ts

```diff
@@ -10,13 +10,11 @@ export class UserService {
   constructor(private httpService: HttpClient) {}
 
   getLoggedInUser(): Observable<UserDTO> {
-    // for now we will get all user, and show first
-    return this.httpService
```

Comment:
@TilegenGena This was nice, but user kept changing. I hard coded with the user id 1.